frontend/components/CategoryServicesDropdown.jsx

```diff
@@ -0,0 +1,27 @@
+import Link from "next/link";
+import styles from "../styles/navBar.module.css";
+
+export default function CategoryServicesDropdown({ category,isOpened }) {
+  return (
+    <span className={`${isOpened ? styles.dropdown_burger : styles.dropdown}`}>
+      <Link href='/'>
+        <a
+          className={`${
+            isOpened ? styles.main_nav_links_burger : styles.main_nav_links
+          }`}
+        >
+          {category.title}
+        </a>
+      </Link>
+      <div
+        className={`${
+          isOpened ? styles.dropdown_content_burger : styles.dropdown_content
+        }`}
+      >
+        {category.services?.map((service) => (
+          <p key={service.title}>{service.title}</p>
```

Comment:
Each item in the dropdown will in the future be a link to a service page 
```suggestion
          <a key={service.title} href="#">{service.title}</a>
```

Reply:
Also, lets make dropdown open by click nor by hover. Because for now it has strange behaviour in the mobile view with touch events